test/e2e/backups/ttl.go

```diff
@@ -122,19 +122,31 @@ func TTLTest() {
 
 		var snapshotCheckPoint SnapshotCheckPoint
 		if useVolumeSnapshots {
-			if veleroCfg.CloudProvider == Vsphere {
-				// TODO - remove after upload progress monitoring is implemented
+			if veleroCfg.HasVspherePlugin {
```

Comment:
trivial, is the `TODO` no longer valid?

Reply:
IMO, the vSphere plugin is not related to the upload progress monitoring.